qase-python-commons/src/qaseio/commons/testops.py

```diff
@@ -466,10 +469,10 @@ def merge_log_execution_attachments(class_result: Result, file_name: str = "log.
             )
 
     @staticmethod
-    def merge_stacktrace(class_result: Result) -> None:
+    def merge_stacktrace(test_result: Result) -> None:
         subtests_stacktrace = [
             subtest_execution.stacktrace
-            for subtest_execution in class_result.class_execution
-            if subtest_execution.stacktrace and subtest_execution.status == "failed"
+            for subtest_execution in test_result.results_to_merge
+            if subtest_execution.stacktrace and subtest_execution.status in ["failed", "rerun"]
```

Comment:
ah, nice catch with rerun status here!

Reply:
Yeah, it was broken for a long time :smile: